man/puppet-query.pod

```diff
@@ -27,6 +27,29 @@ the options specific to that action.
 
 Displays version information
 
+=item B<-c>,B<--config> <path>
+
+Overrides the path for the PuppetDB CLI config.
+Default: ~/.puppetlabs/client-tools/puppetdb.conf
```

Comment:
'See `man puppetdb_conf` for more details on the contents of this file' ? Or something like that?